apis/placement/v1beta1/clusterresourceplacement_types.go

```diff
@@ -448,10 +451,15 @@ type ApplyStrategy struct {
 	// ServerSideApplyConfig defines the configuration for server side apply. It is honored only when type is ServerSideApply.
 	// +optional
 	ServerSideApplyConfig *ServerSideApplyConfig `json:"serverSideApplyConfig,omitempty"`
+
+	// TakeoverAction describes the action to take when we place the selected resources on the target cluster the first time.
+	// +kubebuilder:default=AlwaysApply
+	// +kubebuilder:validation:Enum=AlwaysApply;ApplyIfNoDiff
+	// +optional
+	TakeoverAction TakeOverActionType `json:"actionType,omitempty"`
```

Comment:
Does TakeoverAction have no impact if ApplyStrategyType is set to ReportDiff?

Reply:
yes, the ApplyStrategyType determines the action